internal/cloud/realm/auth.go

```diff
@@ -96,6 +97,24 @@ func (c *client) getAuth(options api.RequestOptions) (string, error) {
 	return "", nil
 }
 
+//Refresh Auth uses the user's refresh token to attempt to create a new access token
```

Comment:
docstring can be omitted, or otherwise it should be cleaned up (i'd just remove it)

```go
// refreshAuth uses the refresh token to create a new access token
```